apps/settings/js/bluetooth.js

```diff
@@ -664,13 +666,14 @@ navigator.mozL10n.once(function bluetoothSettings() {
         }
       } else {
         // show pair process fail.
-        var msg = _('error-pair-title');
+        var l10nId = 'error-pair-title';
         if (errorMessage === 'Repeated Attempts') {
-          msg = msg + '\n' + _('error-pair-toofast');
+          l10nId = 'error-pair-toofast';
         } else if (errorMessage === 'Authentication Failed') {
-          msg = msg + '\n' + _('error-pair-pincode');
+          l10nId = 'error-pair-pincode';
```

Comment:
The reason why I decided to not modify the ID was because previously it would build message like this:
`Unable to pair devices\nUnable to pair with the device. Check that the PIN is correct.`

which I changed to either first line or the latter. Since all messages have the same "Unable to pair with the device" I don't feel like removing the first line changes anything.

Can you look at settings.en-US.properties and let me know what you think?
If you'll still believe I should, I'll change the IDs :)